esm/src/server.js

```diff
@@ -1,16 +1,8 @@
-import http from 'http';
+import mongooseConnect from './db.connect';
 import { app as app } from './app';
-import chalk from 'chalk';
 
 const port = process.env.PORT || 3000;
-const server = http.createServer(app);
 
-/* eslint-disable no-console */
-
-server.listen(port, () => {
-  try {
-    console.log( chalk.cyanBright(`\nServer running at http://localhost:${port}`) );
-  }catch(err) {
-    console.log(err);
-  }
+app.listen(port, () => {
+  mongooseConnect(port);
 });
```

Comment:
Nice work @samuko-things. I see you have the latest changes which make my ongoing clean-up work easier 👍🏽

Please also note the following for the esm template:
- It should also have the .example.env file
- Use the `dev` and `dev:atlas` approach in the package.json file

#

> **Note**
> Do these for the cjs template when you submit a PR for it

Reply:
@samuko-things Please also note the following for the esm template:
- It should also have the .example.env file
- Use the `dev` and `dev:atlas` approach in the package.json file, that way, you will be able to test if it works without issues.

#

> **Note**
> Do these for the cjs template when you submit a PR for it

Reply:
@samuko-things Please also note the following for the esm template:
- It should also have the .example.env file (You can copy the content of the new .example.env file into your own .env file).
- Use the `dev` and `dev:atlas` approach in the package.json file

That way, you will be able to test if it works without issues.

#

> **Note**
> Do these for the cjs template when you submit a PR for it

Reply:
@samuko-things Please also note the following for the esm template:
- It should also have the .example.env file (You can copy the content of the new .example.env file into your own .env file).
- Use the `dev` and `dev:atlas` approach in the package.json file

That way, you will be able to test if it works without issues.

#

> **Note**
> Do these for the cjs template when you submit a PR for it

Reply:
OK. Sorry @samuko-things I did not see the part where you talked about your confusion with package.json file earlier. 

This was what was there previously:

````
"start": "nodemon --exec npm-run-all --parallel open:src lint:watch",
"open:src": "babel-node src/server.js",
````
#

What to change it to... 

1. You have to kind of divide `open:src` into local and atlas first like so:

````
"open:src:local": "babel-node path/to/local/server/here",
"open:src:atlas": "babel-node path/to/atlas/server/here"
````

2. Then divide `start` into `dev` and `dev:atlas` like so:
````
"dev": "nodemon --exec open:src:local",
"dev:atlas": "nodemon --exec open:src:atlas"
````

# 

> **Note**
> - If you notice, we now have a function EslintWatch() somewhere else in the code that takes care of running our lint:watch command. So that's why I no longer added it in the dev or dev:atlas line my code snippet.
> - We also do not need npm-run-all package any more. It helped us to run 2 npm scripts together before, now that we are running Eslint watch from somewhere else, you can safely **uninstall the npm-run-all package**. This is also why I no longer included it too in my code snippet.

#
That said, I think it's shorter and easier to just do it straight like this, and remove the open:src complexity:

````
"dev": "nodemon --exec babel-node path/to/local/server/here",
"dev:atlas": "nodemon --exec babel-node path/to/atlas/server/here"
````

> **Note**
> I can't remember if it's `nodemon - - exec` or just `nodemon` when not using npm-run-all. Just experiment. And let me know if you have any issues while doing so.